package.json

```diff
@@ -28,32 +28,33 @@
     "version": "npm run build && git add dist"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "^4.5.3",
-    "@vue/cli-plugin-e2e-cypress": "~4.5.0",
-    "@vue/cli-plugin-eslint": "^4.5.3",
-    "@vue/cli-plugin-unit-jest": "^4.5.3",
-    "@vue/cli-service": "^4.5.3",
+    "@vue/cli-plugin-babel": "^4.5.9",
+    "@vue/cli-plugin-e2e-cypress": "^4.5.9",
+    "@vue/cli-plugin-eslint": "^4.5.9",
+    "@vue/cli-plugin-unit-jest": "^4.5.9",
+    "@vue/cli-service": "^4.5.9",
     "@vue/eslint-config-airbnb": "^5.0.2",
     "@vue/test-utils": "^1.0.3",
-    "axios": "^0.19.0",
+    "axios": "^0.21.0",
     "babel-eslint": "^10.1.0",
-    "core-js": "^3.6.4",
+    "core-js": "^3.8.1",
     "cross-spawn": "^7.0.1",
     "cypress-xpath": "^1.6.1",
-    "eslint": "^7.6.0",
+    "eslint": "^7.15.0",
     "eslint-plugin-cypress": "^2.11.2",
     "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-vue": "^6.2.2",
+    "eslint-plugin-vue": "^7.2.0",
     "jest-canvas-mock": "^2.2.0",
-    "material-design-icons-iconfont": "^5.0.1",
+    "material-design-icons-iconfont": "^6.1.0",
     "node-sass-glob-importer": "^5.3.2",
     "promise-polyfill": "^8.1.3",
     "replace": "^1.2.0",
-    "sass": "^1.26.5",
-    "sass-loader": "^9.0.3",
+    "sass": "^1.30.0",
+    "sass-loader": "^10.1.0",
     "striptags": "^3.1.1",
     "vue": "^2.6.11",
-    "vue-template-compiler": "^2.6.11"
+    "vue-template-compiler": "^2.6.11",
+    "webpack": "^4.44.2"
```

Comment:
Don't know if it's good idea to include webpack as dep here. Although it makes sense as we use a require of BannerPlugin in vue.config,js. Maybe there is a different solution.